### PR TITLE
Add gas meter readings support

### PR DESCRIPTION
**Pull request by legolasbo**
For my pet project I also need the gas meter readings. So this PR adds support for them to your package :)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,3 +1,5 @@
-module github.com/roaldnefs/go-dsmr
+module github.com/legolasbo/go-dsmr
```
Comment (roaldnefs) [inline go.mod]:
```suggestion
module github.com/roaldnefs/go-dsmr
```

File: telegram_test.go [hunk 2]
```diff
@@ -1,13 +1,18 @@
 package dsmr_test
 
 import (
-	"github.com/roaldnefs/go-dsmr"
+	"github.com/legolasbo/go-dsmr"
```
Comment (roaldnefs) [inline telegram_test.go]:
```suggestion
	"github.com/roaldnefs/go-dsmr"
```

**Review by roaldnefs — CHANGES_REQUESTED** [review]
Thanks @legolasbo for your pull request! We you be able the change the module back to `github.com/roaldnefs/go-dsmr`?

**Comment by legolasbo** [comment]
Those changes didn't feel right to me either, but it seemed like the tests wouldn't pass without them. Reverted the change.

**Comment by legolasbo** [comment]
Is there anything else you want me to change before merging @roaldnefs?

**Comment by roaldnefs** [comment]
Thanks @legolasbo  for your contribution! It took a bit longer before I could test your pull request but I finally found the time.

**Pull request merged**